src/SIL.XForge.Scripture/ClientApp/src/app/shared/share/share-dialog.component.html

```diff
@@ -1,12 +1,90 @@
-    <button id="close-btn" mat-flat-button mat-dialog-close>{{ t("done") }}</button>
+  <h1 mat-dialog-title>
+    <span>{{ t("share") }} '{{ projectName }}'</span
+    ><button mat-icon-button mat-dialog-close tabindex="-1"><mat-icon>close</mat-icon></button>
+  </h1>
+  <div class="configurations">
+    <div class="configuration-option configuration-invitation-role">
+      <mat-icon>public</mat-icon>
+      <div>
+        {{ t(shareLinkType) }} {{ t("can") }} <b>{{ i18n.localizeRoleDescription(shareRole) }}</b>
+        <a [matMenuTriggerFor]="roleMenu" title="{{ t('invitation_role') }}" *ngIf="canUserChangeRole"
+          >({{ t("change") }})</a
+        >
+      </div>
+      <mat-menu #roleMenu>
+        <mat-selection-list [multiple]="false" class="share-selection-role">
+          <mat-list-option *ngFor="let role of availableRoles" [selected]="role === shareRole" (click)="setRole(role)">
+            <div class="role-name">{{ i18n.localizeRole(role) }}</div>
+            <div class="role-description">{{ i18n.localizeRoleDescription(role) }}</div>
+          </mat-list-option>
+        </mat-selection-list>
+      </mat-menu>
+    </div>
+    <div class="configuration-option">
+      <mat-icon>translate</mat-icon>
+      <div>
+        {{ t("invitation_shared_in_language") }} <b>{{ shareLocaleCode.localName }}</b>
+        <a [matMenuTriggerFor]="langMenu" title="{{ t('invitation_language') }}">({{ t("change") }})</a>
+      </div>
+      <mat-menu #langMenu>
+        <mat-selection-list [multiple]="false">
+          <mat-list-option
+            *ngFor="let locale of i18n.locales"
+            [selected]="locale === shareLocaleCode"
+            (click)="setLocale(locale)"
+          >
+            <span [class.locale-disabled]="!locale.production">{{ locale.localName }}</span>
+          </mat-list-option>
+        </mat-selection-list>
+      </mat-menu>
+    </div>
+    <ng-container *ngIf="canUserChangeLinkUsage">
+      <div class="configuration-option configuration-link-type">
+        <mat-icon>groups</mat-icon>
+        <div>
+          {{ t("link_can_be_used_by") }} <b>{{ t(shareLinkType) }}</b>
+          <a
+            [matMenuTriggerFor]="linkUsageMenu"
+            title="{{ t('invitation_link_type') }}"
+            *ngIf="shareLinkUsageOptions.length > 1"
+            >({{ t("change") }})</a
+          >
+        </div>
+        <mat-menu #linkUsageMenu>
+          <mat-selection-list [multiple]="false" *ngIf="shareLinkUsageOptions.length > 1">
+            <mat-list-option
+              *ngFor="let option of shareLinkUsageOptions"
+              [selected]="option === shareLinkType"
+              (click)="setLinkType(option)"
+            >
+              {{ t(option) }}
+            </mat-list-option>
+          </mat-selection-list>
+        </mat-menu>
+      </div>
+      <app-notice *ngIf="isRecipientOnlyLink" icon="info" text="share_dialog.recipient_only_notice"></app-notice>
```

Comment:
I'm afraid this notice does too good of a job at grabbing the user's attention, to the point of distracting for what the user should be looking at (the controls that come ahead of it). Even having seen screenshots of it before, the first time I checked this out I found my eyes physically drawn to it and read it before anything else. What do you think about this concept?

![](https://user-images.githubusercontent.com/6140710/211651858-2d3940c1-69d8-4afb-b5ec-b6b2f9c1c0f5.png)


Also, I think the notice should also tell the user when the link can be re-used:

"This link can be used repeatedly. Anyone you share the link with will be able to share it with another person who will also be able to join the project."

And in the case of one-time-use links, let's add a note that the link expires after 14 days.